pkg/workflowengine/impl/k8s_executor_test.go

```diff
@@ -224,9 +242,17 @@ func TestExecute_MiscError(t *testing.T) {
 		assert.Equal(t, namespace, ns)
 		return &fakeFlyteWorkflow
 	}
+
+	mockApplicationConfig := runtimeMocks.MockApplicationProvider{}
+	mockApplicationConfig.SetTopLevelConfig(runtimeInterfaces.ApplicationConfig{
+		OffloadWorkflowClosureToStorage: false,
```

Comment:
Can we have a unit test covering the true case and asserting on workflowspec, subworkflows and task to be nil.